core/iwasm/libraries/lib-socket/test/build.sh

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
```suggestion
#!/bin/bash

#
# Copyright (C) 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
#
```

Reply:
Also, please add `set -ueo pipefail` so the script fails if any file fails to compile.